modules/backend/lang/it/lang.php

```diff
@@ -6,7 +6,7 @@
     ],
     'field' => [
         'invalid_type' => 'Il tipo di campo :type non è valido.',
-        'options_method_not_exists' => 'La classe :model deve definire un metodo :method() che ritorni le opzioni per il campo ":field".'
+        'options_method_not_exists' => "La classe :model deve definire un metodo :method() che ritorni le opzioni per il campo ':field'.",
```

Comment:
Please revert all the changes that are just quote changes, it unnecessarily clutters the history of the project and makes it more difficult to review the changes as a whole.